.env.template

```diff
@@ -0,0 +1,6 @@
+MERLIN_USERNAME=
```

Comment:
Can we prefix `MERLIN_USERNAME` and `MERLIN_PASSWORD` with `AERIE_` instead of `MERLIN_`? Merlin is a specific service and the since this is a system-wide default I think the name should reflect that.

Reply:
I've done that change! Feel free to resolve this conversation if you're happy with it.